package.json

```diff
@@ -2,6 +2,9 @@
   "name": "go-ipfs-dep",
   "version": "0.4.4",
   "description": "Install the latest go-ipfs binary",
+  "go-ipfs": {
+    "version": "v0.4.4"
+  },
```

Comment:
You can use one or the other. If you don't specify the "go-ipfs.version" field, then the package version is used. This is to enable us to release a) multiple version of the same go-ipfs version (eg. 0.4.5-1) and b) enable us to release dev versions of go-ipfs.